src/language/__tests__/schema-printer-test.js

```diff
@@ -83,6 +83,8 @@ union Feed = Story | Article | Advert
 
 union AnnotatedUnion @onUnion = A | B
 
+union AnnotatedUnionTwo @onUnion = A | B
```

Comment:
Nope the printer will only print without it or with it I think it's better to leave it out and add it with prettier for ex